src/user/test/mock/user.model.mock.ts

```diff
@@ -0,0 +1,57 @@
+import { UserCredentialsDto } from '../../dto/user.credentials.dto';
+import { faker } from '@faker-js/faker';
+import { User } from '../../user.schema';
+import { UserDto } from '../../dto/user.from.db.dto';
+
+export const mockUsername = faker.name.firstName();
+export const mockPassword = faker.internet.password(10, false, undefined, 'aA$1');
+export const mockId = faker.database.mongodbObjectId();
+export const mockCredentials: UserCredentialsDto = { username: mockUsername, password: mockPassword };
```

Comment:
try to aggregate those mocks into appropriate groups, in my approach it would look like 
`const mockUsername = faker.name.firstName();`
`const mockPassword = faker.internet.password(10, false, undefined, 'aA$1');`
`const mockId = faker.database.mongodbObjectId();`
`const mockCredentials: UserCredentialsDto = { username: mockUsername, password: mockPassword };`
`export const VALUES = {
mockUsername,
mockPassword,
mockId,
mockCredentials
};`

It is strictly personal choice but I believe it to make exporting much cleaner

Reply:
I don't really see a point in doing that. After importing VALUES you still have to either destructure it or use VALUES.mockUsername.